app/Http/Controllers/BookingRequestController.php

```diff
@@ -22,7 +22,7 @@ public function index()
     {
         return inertia('Admin/BookingRequests/Index', [
             'booking_requests' => BookingRequest::with('user', 'room')->get(),
-            'rooms' => Room::all(),
+            'rooms' => Room::hideUserRestrictions(auth()->user())->get(),
```

Comment:
should probably pull user off of $request->user rather than relying on a global method.